packages/remix-dev/vite/plugin.ts

```diff
@@ -1001,12 +1015,16 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
             );
           }
 
-          if (pluginConfig.isSpaMode) {
+          if (isSpaMode) {
             await handleSpaMode(
+              // TODO: Is this readily available anywhere?
+              await ssrBuildContext.getManifest(),
```

Comment:
@pcattori @markdalgleish Is it an issue to generate this on demand like this?  Is there a cached version anywhere I can access?